org-eclipse-che-jdt-ls-extension/pom.xml

```diff
@@ -79,12 +80,21 @@
                                 <skip>true</skip>
                             </configuration>
                         </execution>
+                        <execution>
```

Comment:
Why do we need this and what does it do?

Reply:
on release phase maven runs extra checks, one of them is check java  versions compatibility (it is set in che-parent) it was failing so I skipped it because I have no time to work on this.

Reply:
btw, idk if it should be updated manually for next version should it now be set to 0.0.2 ?
I never used this plugin

Reply:
there is an issue actually, after release `mvn clean install` now fails because it does not forces this check
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M1:enforce (check-java-compat) on project org-eclipse-che-jdt-ls-extension: The parameters 'rules' for goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M1:enforce are missing or invalid -> [Help 1]
```

Reply:
this is fixed now with https://github.com/eclipse/che-parent/commit/ba0b97d7807044cba7f5dbb86493d332ae40f1f8